contracts/liquidityMining/KyberFairLaunch.sol

```diff
@@ -347,39 +325,40 @@ contract KyberFairLaunch is IKyberFairLaunch, PermissionAdmin, ReentrancyGuard {
   }
 
   /**
-  * @dev harvest reward from pool for the sender
-  * @param _pid: id of the pool
-  */
+   * @dev harvest reward from pool for the sender
+   * @param _pid: id of the pool
+   */
   function harvest(uint256 _pid) public override {
     updatePoolRewards(_pid);
     _updateUserReward(msg.sender, _pid, true);
   }
 
   /**
-  * @dev update reward for one pool
-  */
+   * @dev update reward for one pool
+   */
   function updatePoolRewards(uint256 _pid) public override {
     require(_pid < poolLength, 'invalid pool id');
     PoolInfo storage pool = poolInfo[_pid];
     uint32 lastAccountedBlock = _lastAccountedRewardBlock(_pid);
-    if (lastAccountedBlock <= pool.lastRewardBlock || lastAccountedBlock < pool.startBlock) return;
```

Comment:
If pool is renewed with a later startBlock, then we don't / shouldn't updatePoolRewards, so should still keep this condition

Reply:
when pool is renewed, the  `lastAccountedBlock ` is updated to new start block